frontend/pages/topics/index.tsx

```diff
@@ -48,7 +48,7 @@ const Topics: FC<TopicsProps> = ({ topicsData }: TopicsProps) => {
                       border: ".08rem solid black",
                     }}
                   >
-                    {topic.name}
+                    {`${topic.name} (${Object.keys(topic.propositions).length})`}
```

Comment:
I think the length retrieval here can be simplified to `topic.propositions.length`. Since `topic.proposition` is an array, `Object.keys(topic.propositions)` returns an array of indexes, like `[0, 1, 2]`.